homeassistant/components/logbook/__init__.py

```diff
@@ -528,8 +600,27 @@ def yield_rows(query: Query) -> Generator[Row, None, None]:
         #
         return query.yield_per(1024)  # type: ignore[no-any-return]
 
+    json_quotable_entity_ids = None
+    if entity_ids:
+        # Must not be the same object as entity_ids
+        # since sqlalchemy caches on the object
+        json_quotable_entity_ids = list(entity_ids)
+    json_quotable_device_ids = None
+    if device_ids:
+        # Must not be the same object as device_ids
+        # since sqlalchemy caches on the object
+        json_quotable_device_ids = list(device_ids)
```

Comment:
I just realized I can move these one level deeper since they only need to be created before they are passed to the closure that creates the lambda_stmt

Reply:
moved in 97c4c6bd666199792ea8c0f98f83b2adc56770f2